source/directives/field-label.ts

```diff
@@ -86,6 +92,10 @@ module formFor {
     link($scope:FieldLabelScope, $element:ng.IAugmentedJQuery, $attributes:ng.IAttributes):void {
       $scope.attributes = $attributes;
 
+      $scope.$watch('useGlyphicons', function(useGlyphicons) {
```

Comment:
Makes sense! I was unsure what `useGlyphicons: '@'` did and could not find any documentation to what `'@'` actually does.